<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>216</width>
     <height>196</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_register">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_input">
      <item row="0" column="0">
       <widget class="QLabel" name="label_userName">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="userName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nickName">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="nickName">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_passWord">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="passWord">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_confirmpassWord">
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="confirmpassWord">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_userType">
        <property name="text">
         <string>用户类型</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radioButton_player">
        <property name="text">
         <string>闯关者</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="radioButton_manager">
        <property name="text">
         <string>出题者</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_button">
      <item row="0" column="0">
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>确认注册</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>返回登录界面</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>40</y>
     <width>270</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt;</string>
   </property>
   <property name="text">
    <string>单词消除游戏用户注册</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
